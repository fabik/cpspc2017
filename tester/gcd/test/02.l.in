15
10 10 21 15 15 17 20 10 26 6 17 9 12 14 6
