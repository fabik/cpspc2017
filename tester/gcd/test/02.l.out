1
27
48
68
85
102
117
132
146
158
170
182
192
202
208
